docs/source/basic/library.rst

```diff
@@ -107,6 +107,32 @@ The kernel function object is shared across all threads in all blocks.
 Due to the block execution order being undefined, there is no safe and consistent way of altering state that is stored inside of the function object.
 Therefore, the ``operator()`` of the kernel function object has to be ``const`` and is not allowed to modify any of the object members.
 
+Kernels can also be defined via lambda.
+
+.. code-block:: cpp
+
+   auto kernel = [] ALPAKA_FN_ACC (auto const & acc /* , ... */) -> void {
+	// ...
+   }
+
+.. attention::
+   The Nvidia ``nvcc`` does not support generic lambdas.
```

Comment:
```suggestion
   The Nvidia ``nvcc`` does not support generic lambdas which are marked with `__device__`, which is what `ALPAKA_FN_ACC` expands to (among others) when the CUDA backend is active.
```

The destinction is important, because you can of course have generic lambdas everywhere else :)